src/rules/__tests__/no-large-snapshots.test.js

```diff
@@ -78,15 +79,77 @@ describe('no-large-snapshots', () => {
     });
   });
 
-  describe('ExpressionStatement function', () => {
+  describe('toMatchSnapshot expression statement and call expressions', () => {
+    const testNode = {
```

Comment:
Can we maybe use ESLint's parser here to write these nodes as source code template literals in the tests instead? Would keep the file from exploding in size

Reply:
Would also make it a lot easier to see if we have all cases we can think of covered with tests

Reply:
@jeysal added a test for snapshot in a describe, and multiple tests w/ mulitple snaps.  let me know if you can think of others.